src/main/java/org/prebid/server/events/EventUtil.java

```diff
@@ -18,6 +19,7 @@
     private static final String IMP_TYPE = "imp";
 
     private static final String BID_ID_PARAMETER = "b";
+    private static final String BIDDER_PARAMETER = "bidder";
```

Comment:
Dot you need to update TEMPLATE_URL to use ts and bidder parameters?

And `toUrl` method